setup-hakn/README.md

```diff
@@ -0,0 +1,57 @@
+# Setup Knative
```

Comment:
Setup Hakn Knative [Serving]

not sure if serving is needed.
is the plan to include eventing in hakn at some point?

Reply:
Done.

Yes we will include Eventing as well later.